packages/terraform-modules-public/gcp/example/variables.tf

```diff
@@ -169,7 +169,7 @@ variable attestation_service_docker_image {
 
   default = {
     repository = "us.gcr.io/celo-testnet/celo-monorepo"
-    tag        = "attestation-service-mainnet"
+    tag        = "attestation-service-1-0-2"
```

Comment:
Could we actually update to 1.0.3? (announced here https://discordapp.com/channels/600834479145353243/653752964627955752/750044099725361173)